packages/wdio-cli/src/interface.js

```diff
@@ -213,95 +255,110 @@ export default class WDIOCLInterface extends EventEmitter {
-            this.interface.log(chalk.bgRed.black('Stderr:\n') + buffer.join(''))
+            const buffer = this.stderrBuffer.slice(bufferLength - maxBufferLength)
+            this.display.push(chalk.bgRed.black('Stderr:\n') + buffer.join(''))
         }
+
         if (this.messages.worker.error) {
             const bufferLength = this.messages.worker.error.length
             const maxBufferLength = !length || length > bufferLength ? bufferLength : length
             const buffer = this.messages.worker.error.slice(bufferLength - maxBufferLength)
-            this.interface.log(chalk.bgRed.black('Worker Error:\n') + buffer.map(
+            this.display.push(chalk.bgRed.black('Worker Error:\n') + buffer.map(
                 (e) => e.stack
-            ).join('\n') + '\n')
+            ).join('\n'))
         }
     }
 
     printSummary() {
         const totalJobs = this.totalWorkerCnt - this.result.retries
-
-        return this.interface.log(
+        return this.display.push([
             'Test Suites:\t', chalk.green(this.result.passed, 'passed') + ', ' +
             (this.result.retries ? chalk.yellow(this.result.retries, 'retries') + ', ' : '') +
             (this.result.failed ? chalk.red(this.result.failed, 'failed') + ', ' : '') +
             totalJobs, 'total',
             `(${totalJobs ? Math.round((this.result.finished / totalJobs) * 100) : 0}% completed)`
-        )
+        ].join(' '))
     }
 
     updateClock (interval = 100) {
         const clockSpinnerSymbol = this.getClockSymbol()
 
-        clearTimeout(this.interval)
-        this.interface.clearLine()
-        this.interface.write('Time:\t\t ' + clockSpinnerSymbol + ' ' + ((Date.now() - this.start) / 1000).toFixed(2) + 's')
+        this.resetClock()
+
+        /**
+         * clear time row if given
+         */
+        if (this.display.length && this.display[this.display.length - 1].startsWith('Time:')) {
+            this.display.pop()
+        }
+
+        this.display.push('Time:\t\t ' + clockSpinnerSymbol + ' ' + ((Date.now() - this.start) / 1000).toFixed(2) + 's')
         this.interval = setTimeout(() => this.updateClock(interval), interval)
+        logUpdate(this.display.join('\n'))
     }
 
     getClockSymbol () {
         return clockSpinner.frames[this.clockTimer = ++this.clockTimer % clockSpinner.frames.length]
     }
 
-    reset () {
+    resetClock () {
         clearTimeout(this.interval)
-        this.interface.log('\n')
-
-        if (this.isWatchMode) {
-            return
-        }
-
-        this.interface.reset()
     }
 
     finalise () {
-        this.interface.clearAll()
-        this.printStdout()
+        this.clearConsole()
         this.printReporters()
+        this.printStdout()
         this.printSummary()
         this.updateClock()
-        this.reset()
+        this.resetClock()
+
+        // eslint-disable-next-line
+        console.log('')
```

Comment:
what does this stand for?

Reply:
it adds a line break at the end of the final report, added a comment